bokehjs/src/coffee/core/util/wheel.ts

```diff
@@ -7,7 +7,11 @@
  */
 
 function fontSize(element: Element): number | null {
-  return parseInt(getComputedStyle(element).fontSize, 10) || null
+  const value = getComputedStyle(element).fontSize
+  if (value != null)
+    return parseInt(value, 10)
+  else
+    return null
```

Comment:
I'm assuming there's not some weirdo JS reason a direct comparison against `null` will fail